src/reducers/filterReducer.js

```diff
@@ -0,0 +1,20 @@
+import { CHANGE_FILTER } from '../actions';
+
+const initialState = {
+  filter: '',
+};
+
+const changeFilterReducer = (state = initialState, action) => {
```

Comment:
Oops, we need to set the state to `All`, that is; users are meant to see all books at first load.
![image](https://user-images.githubusercontent.com/31897434/124323013-7034d080-db78-11eb-88af-617a9db19f14.png)

Reply:
Good job guys